### DISCUSSION TITLE
suggested a 2nd smaller 'Settings' window with essentials only

### DISCUSSION THREAD

**Discussion by kezzy1966** [jamulussoftware/jamulus#1033] [UI Related]
Hi All,

In trying to get the best performance out of Jamulus I'm finding that I now always have the 'Settings' window opened during a session and it is the first thing I get others to open if there are problems. 

However the window is a bit large and really I think we are only interested in:
Buffer size,
Jitter (local+server) as just a number,
Maybe the jitter buffer green/ red indicator (maybe one for each ?)
Auto jitter toggle switch,
Ping Time,
Overall Delay. 

The rest of the fields are valid but rarely looked at once set.

Is there any chance this small window could be an option? 
Maybe even just a single line of the figures ? 
Or even better to have it pop up as a line on the Jamulus main console ?

**Comment by pljones** [comment]
Interesting idea -- a "Status" window with (some of) those settings would take up less space and I think a lot of people probably do run with settings open just to keep an eye on the status indicators.

> Buffer size,

Not so much a status.  I also view this as a "Once it's set correctly, forget it" value.

> Maybe the jitter buffer green/ red indicator (maybe one for each ?)

It's been mentioned elsewhere (https://github.com/corrados/jamulus/issues/747#issuecomment-735892060) that only the local failures are seen (the server failures happen on the server and the server doesn't currently tell the client about them).

> Auto jitter toggle switch,

If you're going to switch it, you probably need the sliders to make adjustments, so leave the switch with the sliders.

**Comment by nico-dr** [comment]
This sounds for me we looking for a diagnostic only dialog. 

Yes, we need more diagnostic information that people can help themself and we can easier help them. My expierence is that it is helpful to combine diagnostics direct with the possibility to make changes so you can direct verify the result of the new setting in the same dialog.

So I agree to a "smaller Settings (/ Diagnostic)  Window" for things you can't change (ping time) and some combined results (like overall delay). Not the buffer size, but the latency of the audio interface (also there is a direct relation to the buffer size). No auto jitter toggle switch. but the measured ping jitter, ...

In my opinion can this information also be displayed i.e. in a status bar of the main (with the possibility to hide it)

**Comment by gilgongo** [comment]
@nico-dr 

> it is helpful to combine diagnostics direct with the possibility to make changes so you can direct verify the result of the new setting

Indeed - but unfortunately I don't think that's possible (as far as I understand): https://github.com/corrados/jamulus/issues/762#issuecomment-741960972

There is no useful relationship between a programmatic state inside Jamulus and hearing, say, a burbling sound or a cut in the audio.

**Comment by corrados** [comment]
BTW: Is the settings window also part of https://github.com/corrados/jamulus/issues/787?

**Comment by gilgongo** [comment]
Hi all - so that we can round up all UI-related idea into a wider discussion before speccing out what we'd like to do, I'll move this to a Discussion for now if that's OK.

**Comment by pljones** [comment]
I guess a lot of this will now have been addressed by the various changes that have been made.
